controller/users/list.go

```diff
@@ -36,6 +38,21 @@ func HandleList(w http.ResponseWriter, r *http.Request) error {
 		return err
 	}
 
-	controller.EncodeJSONResponse(w, userviews.ListFrom(users))
+	groupId, err := usergroups.GetGroupIDFrom(r)
+	if err != nil {
+		logrus.Error(err)
+		return err
+	}
+
+	roles := make([]groupenums.Role, len(users))
+	for i, user := range users {
+		roles[i], err = model.GetUserRoleByID(db, user.ID, *groupId)
+		if err != nil {
+			logrus.Error(err)
+			return err
+		}
+	}
```

Comment:
N + 1 query problem

Reply:
Filed as an issue: #121.

We can just merge first as it's not critical at the moment.